logger/old_logger.go

```diff
@@ -0,0 +1,21 @@
+package logger
+
+import (
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/sonatype-nexus-community/nancy/types"
+)
+
+// GetLogFileLocation will return the location on disk of the log file
+func GetLogFileLocation() (result string) {
```

Comment:
Is the purpose of the function rename and new `old_logger` to maintain backwards compatibility? If so, I'm not sure it's worth the overhead. Also, is it true that `old_logger` is never actually used now? If so, it seems like more maintenance burden (who will promise to remember to delete it?) If you decide to keep it, should it be marked as deprecated?